### ISSUE TITLE
Release fine-tuned models from Learning to Optimize Multi-Objective Alignment Through Dynamic Reward Weighting on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [yining610/dynamic-reward-weighting#1]
Hi @ylu610 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2509.11452.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw in your GitHub README that you noted: "we need to merge the saved checkpoints from FSDP and Megatron backends to HuggingFace models first." It'd be great to make these fine-tuned LLM checkpoints, embodying your dynamic reward weighting (e.g., Qwen3-8B, Deepseek-7B), available on the 🤗 Hub! This would greatly improve their discoverability and visibility. We can add tags to the model cards so that people find the models easier, link them to the paper page, etc.

If you're interested, here's a guide: https://huggingface.co/docs/hub/models-uploading. If it's a custom PyTorch model, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to the model, which lets you to upload the model and people to download and use models right away.
If you prefer to upload models through the UI or via other methods, people can also use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

You can also build a demo for your model on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested/need any guidance :)

Kind regards,

Niels

**Comment by yining610** [comment]
Thank you for the suggestions. Unfortunately, we didn't save any checkpoints from the training but used WandB to log the validation results on the fly.
